financial-connections/consumer-rules.txt

```diff
@@ -13,3 +13,25 @@
 -keep,allowobfuscation class ** implements com.airbnb.mvrx.MavericksViewModelFactory
 
 -dontwarn com.google.crypto.tink.subtle.XChaCha20Poly1305
+
+
+# MavericksViewModel loads the Companion class via reflection and thus we need to make sure we keep
+# the name of the Companion object.
+-keepclassmembers class ** extends com.airbnb.mvrx.MavericksViewModel {
```

Comment:
Can we upstream this? Ideally we wouldn't need to add these ourselves.

Reply:
I'll open an issue on Mavericks!